src/medCore/gui/medAbstractWorkspace.cpp

```diff
@@ -569,13 +569,28 @@ void medAbstractWorkspace::buildTemporaryPool()
 
 void medAbstractWorkspace::open(const medDataIndex &index)
 {
+    // Called after right click->View on a series/study, or double-click on a series
     QList<QUuid> containersSelected = d->viewContainerStack->containersSelected();
     if(containersSelected.size() != 1)
         return;
 
     medViewContainer *container = medViewContainerManager::instance()->container(containersSelected.first());
     if(index.isValidForSeries())
+    {
         container->addData(medDataManager::instance()->retrieveData(index));
+    }
+    else if(index.isValidForStudy())
+    {
+        // We get the list of each series from that study index, and open it
+        QList<medDataIndex> seriesList = medDataManager::instance()->getSeriesListFromStudy(index);
+        if (seriesList.count() > 0)
```

Comment:
When dropping a study containing more than 10 (5?) elements, maybe you should display a warning to the user, asking him if he really wants to drop the study.
Because a misdrop (from now on, this word exists) might kill the user's computer.